### PR TITLE
[Naitei] Thanh toán stripe

### PR DESCRIPTION
**Pull request by SuKay712**
## Related Tickets
- ticket redmine

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

https://github.com/user-attachments/assets/10c60634-c174-4ac6-9251-ce7fc1df49ab

### CODE REVIEW

File: app/assets/stylesheets/product.scss [hunk 1]
```diff
@@ -192,3 +193,157 @@
 .create-form_component{
   display: block;
 }
+
+/* Checkout */
+.check-out__body {
+  margin: 32px 160px 40px 160px;
+  color: #1A1A1A;
```
Comment (TranMinhHuy2003) [inline app/assets/stylesheets/product.scss]: đặt tên biến màu nha

File: app/assets/stylesheets/product.scss [hunk 2]
```diff
@@ -192,3 +193,157 @@
...
+
+.check-out__body-left__item p {
+  margin-bottom: 10px;
+}
+
+.check-out__body-left__item input,
+.check-out__body-left__item select,
+.check-out__body-left__item textarea {
+  padding: 14px 0 14px 16px;
+  border: 1px solid #E6E6E6;
```
Comment (TranMinhHuy2003) [inline app/assets/stylesheets/product.scss]: đây nữa nha

File: app/assets/stylesheets/product.scss [hunk 3]
```diff
@@ -192,3 +193,157 @@
...
+  align-items: center;
+  margin-bottom: 10px;
+}
+
+.check-out__body-right__payment-item input {
+  appearance: none;
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  border: 1px solid #E6E6E6;
```
Comment (TranMinhHuy2003) [inline app/assets/stylesheets/product.scss]: đặt biến màu trong đoạn này nữa nha

File: app/models/bill.rb [hunk 4]
```diff
@@ -1,4 +1,7 @@
 class Bill < ApplicationRecord
+  PERMITTED_ATTRIBUTES = [:user_id, :address, :phone_number,
+                          :note_content, :status].freeze
```
Comment (ThaiKhacDuoc) [inline app/models/bill.rb]: dùng %i huynh ơi
```suggestion
  PERMITTED_ATTRIBUTES = %i(user_id address phone_number note_content status).freeze
```

**Review by ThaiKhacDuoc — COMMENTED** [review]
cmt

**Comment by SuKay712** [comment]
ready

File: app/models/bill.rb (hunk 4 above)
Comment (SuKay712) [reply app/models/bill.rb]: oke

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**